### ISSUE TITLE
feat: `no-linebreak-after-decorators`

### ISSUE THREAD

**Issue by alan-agius4** [palantir/tslint#3682]
I'd like a new lint rule. 
#### TypeScript code being linted `no-linebreak-after-decorators` or something along these lines.

The rule should emit an error when there is a breakline after a decorator

Valid 
```ts
@bar()
export foo(): boolean {
 return true;
}
```

Invalid
```ts
@bar()

export foo(): boolean {
 return true;
}
```

**Comment by JoshuaKGoldberg** [comment]
Could this be a part of `whitespace` instead of its own rule? Discussion here is likely a subset of #3548.

**Comment by JoshuaKGoldberg** [comment]
☠️ *TSLint's time has come!* ☠️ 

TSLint is no longer accepting most feature requests per #4534. See **[typescript-eslint.io](https://typescript-eslint.io)** for the new, shiny way to lint your TypeScript code with **ESLint**. ✨ 

It was a pleasure open sourcing with you all!

**Comment by JoshuaKGoldberg** [comment]
🤖 Beep boop! 👉  **TSLint is deprecated** 👈 _(#4534)_  and you should **switch to [typescript-eslint](https://typescript-eslint.io)**! 🤖 

🔒 This issue is being locked to prevent further unnecessary discussions. Thank you! 👋
